shared/hashutil/hash_test.go

```diff
@@ -0,0 +1,17 @@
+package hashutil
```

Comment:
Please black box this test by using package hashutil_test and importing hashutil